### ISSUE TITLE
Consolidate the Zowe client log files into the same directory

### ISSUE THREAD

**Issue by gejohnston** [zowe/zowe-cli#2116]
**Is your feature or enhancement request related to a problem or limitation? Please describe**

The location of our log files is unintuitive. Our 2 log files are in two separate directories. It is not clear that to find a log file that you must traverse into a directory named '.zowe/imperative' or a directory named '.zowe/zowe'

```
C:\USERS\YouUserName\.ZOWE
│
├───imperative
│   └───logs
│       imperative.log
│
└───zowe
    └───logs
        zowe.log
```

**Describe your enhancement idea**

Consolidate the log files into a directory named '.zowe/logs'. Remove the otherwise empty directories named '.zowe/imperative' and '.zowe/zowe'.

```
C:\USERS\YouUserName\.ZOWE
│
└────logs
     imperative.log
     zowe.log
```

**Describe alternatives you've considered**

**Provide any additional context**

Because the product works ok as-is, this is a low priority. It is possible that changing the location of log files could be considered a breaking change and would have to be completed before V3 if the work is undertaken.
